### PR TITLE
buildbot: 0.9.0rc3 -> 0.9.0rc4

### PR DESCRIPTION
**Pull request by nand0p**
buildbot: 0.9.0rc3 -> 0.9.0rc4
- upgrade to rc4
- un-hardcode path in logwatcher
- fix shebang in wrapper
- add gpl2 license
- remove duplicate wrapper

### CODE REVIEW

**Comment by nand0p** [comment]
@FRidh @copumpkin

File: pkgs/development/tools/build-managers/buildbot/default.nix [hunk 1]
```diff
@@ -61,22 +57,26 @@ pythonPackages.buildPythonApplication (rec {
   preInstall = ''
     # writes out a file that can't be read properly
     sed -i.bak -e '69,84d' buildbot/test/unit/test_www_config.py
+
+    # un-hardcode path to tail
+    sed -i.bak 's|/usr/bin/tail|tail|' buildbot/scripts/logwatcher.py
```
Comment (copumpkin) [inline pkgs/development/tools/build-managers/buildbot/default.nix]: I'd actually just make this into `${coreutils}/bin/tail`

Comment (nand0p) [inline pkgs/development/tools/build-managers/buildbot/default.nix]: fixed

File: pkgs/development/tools/build-managers/buildbot/default.nix [hunk 2]
```diff
@@ -61,22 +57,26 @@ pythonPackages.buildPythonApplication (rec {
...
   '';
 
   postFixup = ''
-    buildPythonPath "$out"
```
Comment (nand0p) [inline pkgs/development/tools/build-managers/buildbot/default.nix]: this pr is updating the existing wrapper with the correct shebang.

im not sure why the `PYTHONPATH` is not being set properly by `buildPythonApplication`, but without this wrapper, buildbot does not start up. it just hangs with no output.  this is related to https://github.com/NixOS/nixpkgs/pull/18667

Comment (FRidh) [inline pkgs/development/tools/build-managers/buildbot/default.nix]: I don't understand what you mean here. Is this still needed, yes or no?

Comment (nand0p) [inline pkgs/development/tools/build-managers/buildbot/default.nix]: yes, this is needed.

**Comment by nand0p** [comment]
@FRidh @copumpkin this PR should be ready for review / merge

**Comment by FRidh** [comment]
I still don't get why the wrapper is made like this, but if it works for you then that's fine.

**Pull request merged**